### PR TITLE
2337 adult trapping dashboard strive

### PR DESCRIPTION
**Pull request by billli0**
### Issue 
#2337](https://github.com/beyondessential/tupaia-backlog/issues/2337)
### Changes:
two simple migrations

### CODE REVIEW

File: packages/database/src/migrations/20210524111255-StriveAdultTrappingReportConfig-modifies-data.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            dataSourceEntityType: 'facility',
+          },
+        },
+      ],
+      dataElements: ['STRVEC_AE-AT13'],
+    },
+    transform: [
+      {
+        transform: 'filter',
+        where: "exists($row['value'],$row['dataElement'])",
```
Comment (rohan-bes) [inline packages/database/src/migrations/20210524111255-StriveAdultTrappingReportConfig-modifies-data.js:42]: Does `exists()` support multiple arguments like this? Perhaps it should... could be good...

Regardless, do we need this transform? I think it's safe to assume that every analytic has a `value` and a `dataElement` 👍

File: packages/database/src/migrations/20210524111255-StriveAdultTrappingReportConfig-modifies-data.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      },
+      {
+        transform: 'select',
+        '...': ['value', 'dataElement'],
+      },
+      {
+        transform: 'aggregate',
+        value: 'group',
+        dataElement: 'count',
+      },
```
Comment (rohan-bes) [inline packages/database/src/migrations/20210524111255-StriveAdultTrappingReportConfig-modifies-data.js:52]: You can actually remove fields from the rows during 'aggregate' like this: 
```suggestion
      {
        transform: 'aggregate',
        value: 'group',
        dataElement: 'count',
        '...': 'drop',
      },
```

**Review by rohan-bes — CHANGES_REQUESTED** [review]
Looking good! Just a couple of suggestions on how you can optimise the report transforms 👍

File: packages/database/src/migrations/20210524111255-StriveAdultTrappingReportConfig-modifies-data.js (hunk 2 above)
Comment (billli0) [reply packages/database/src/migrations/20210524111255-StriveAdultTrappingReportConfig-modifies-data.js:52]: That is handy!

File: packages/database/src/migrations/20210524111255-StriveAdultTrappingReportConfig-modifies-data.js (hunk 1 above)
Comment (billli0) [reply packages/database/src/migrations/20210524111255-StriveAdultTrappingReportConfig-modifies-data.js:42]: happy to drop this.

**Review by rohan-bes — APPROVED** [review]
Nice! looks good

**Pull request merged**